script/_dhcpv6_pd_ref

```diff
@@ -30,28 +30,67 @@
 #       This script manipulates DHCPv6-PD-REF configuration.
 #
 
-customise_dhcpcd_conf()
+DHCPCD_ENTER_HOOK="/etc/dhcpcd.enter-hook"
+DHCPCD_EXIT_HOOK="/etc/dhcpcd.exit-hook"
+
+PD_DAEMON_PATH="/opt/pd-daemon"
+PD_DAEMON="${PD_DAEMON_PATH}/dhcp6_pd_daemon.py"
+PD_DAEMON_SERVICE_NAME="dhcp6_pd_daemon.service"
+PD_DAEMON_SERVICE="/etc/systemd/system/${PD_DAEMON_SERVICE_NAME}"
+
+create_custom_dhcpcd_conf()
 {
-    # This has to be run after script/_border_routing, and this will
-    # invalidate all changes to dhcpcd.conf made by script/_border_routing.
-    sudo tee /etc/dhcpcd.conf >/dev/null <<EOF
+    sudo tee /etc/dhcpcd.conf.custom >/dev/null <<EOF
```

Comment:
I would suggest 1 since we are providing a reference device.

For users building their own BRs, I think the infrastructure like dhcpcd might be highly configured and they could build their own daemon by starting from the script below.